PosteBackend/tests/test_models.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+
+from PosteAPI.models import Folder, Post, User
+
+
+class UserModelTest(TestCase):
+    def test_create_user(self):
+        user = User.objects.create(
+            email="test@example.com", password="securepassword123"
+        )
+        self.assertEqual(user.email, "test@example.com")
+        self.assertEqual(user.username, "test@example.com")
+
```

Comment:
```suggestion
self.assertEqual(user.password, "securepassword123")
```

Reply:
Good find -- tired eyes on my part. I made a change to how the test user was created, as we shouldn't be able to pull the user's password like this (it should be hashed). By using Django's user.set_password rather than including it in the create method, we ensure the password is hashed.